packages/amplify-category-function/API.md

```diff
@@ -4,12 +4,12 @@
 
 ```ts
 
-import { $TSAny } from 'amplify-cli-core';
-import { $TSContext } from 'amplify-cli-core';
-import { $TSObject } from 'amplify-cli-core';
+import { $TSAny } from '@aws-amplify/amplify-cli-core';
+import { $TSContext } from '@aws-amplify/amplify-cli-core';
+import { $TSObject } from '@aws-amplify/amplify-cli-core';
 import { BuildType } from '@aws-amplify/amplify-function-plugin-interface';
 import { FunctionParameters } from '@aws-amplify/amplify-function-plugin-interface';
-import { ResourceTuple } from 'amplify-cli-core';
+import { ResourceTuple } from '@aws-amplify/amplify-cli-core';
```

Comment:
Perhaps it's technically beyond the scope of this PR, but this seems like as good a time as any to combine all of these core imports into one statement

Reply:
This is API.md - autogenerated.
We did consolidation in other files, see @evcodes comments above ^.

Reply:
yeah just noticed that. my bad